### PR TITLE
Solutions Added in CP

### PR DESCRIPTION
**Pull request by mechiragjain**
Please read [how to contribute](CONTRIBUTING.md) first!
# Description
Added solution of "Remove Loop from a Linked List" in CPP Module.

### CODE REVIEW

**Comment by anushkrishnav** [comment]
kindly alter the README to link your work and issue a PR . Good day

File: C++/cp/Remove_Loop_in_Linked_List.cpp [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    {
+        data = val;
+        next = NULL;
+    }
+};
+
+void loopHere(Node* head, Node* tail, int position)
+{
+    if(position==0) return;
+
```
Comment (atarax665) [inline C++/cp/Remove_Loop_in_Linked_List.cpp]: Remove these unnecessary lines.

File: C++/cp/Remove_Loop_in_Linked_List.cpp [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+        next = NULL;
+    }
+};
+
+void loopHere(Node* head, Node* tail, int position)
+{
+    if(position==0) return;
+    
+    Node* walk = head;
+    for(int i=1; i<position; i++)
```
Comment (atarax665) [inline C++/cp/Remove_Loop_in_Linked_List.cpp]: As per good coding habit space should be added before and after operators.

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Comment by atarax665** [comment]
@mechiragjain Make the suggested changes and make sure that you change the README to link your work.

File: C++/cp/Remove_Loop_in_Linked_List.cpp [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+    if ( isLoop(head) || length(head) != n )
+      cout << "0\n";
+    else
+      cout << "1\n";
+  }
+  return 0;
+}
+
+
+void removeLoop(Node* head)
```
Comment (atarax665) [inline C++/cp/Remove_Loop_in_Linked_List.cpp:112]: Shift this function above the driver code and use proper comments where necessary.

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Comment by atarax665** [comment]
And make changes to the README.md file.

**Comment by atarax665** [comment]
The PR is being closed due to inactivity from contributor's end.

**Pull request closed without merging**